airflow/config_templates/airflow_local_settings.py

```diff
@@ -172,6 +196,20 @@
 
 REMOTE_LOGGING = conf.get('core', 'remote_logging')
 
+if os.environ.get('CONFIG_PROCESSOR_MANAGER_LOGGER') == 'True':
```

Comment:
Hi @yrqls21 , may you advise what this environment variable `CONFIG_PROCESSOR_MANAGER_LOGGER` is meant to be? Is it going to be specified by the user, or it's environment-specific?

It's not documented anywhere else.

Reply:
I have noticed the relevant chunk in `airflow/utils/dag_processing.py`... So this environment variable is some sort of "flag" to help determine if these a few following lines here in `airflow/config_templates/airflow_local_settings.py` should be run when we load/reload `airflow_local_settings`, if I'm not mistaken?

Reply:
This feels like it should be a config option somewhere in the config, and then the normal `AIRFLOW__${SECT}__${OPT}` environment variables could be used.

Also there is zero documentation or metion anywher eelse what this block is doing.  Adding to the default config (with comments explaining it's use) gives us two birds with one stone.

Reply:
@XD-DENG you are perfectly correctly. The purpose is to avoid `RotatingFileHandler` to be initialized in multiple processes, which would cause exceptions. I thought I had some comments around it but obviously I'm wrong :(

@ashb This is more like a internal flag controlling the behavior of different processes, similar idea as this line https://github.com/apache/incubator-airflow/blob/master/airflow/bin/cli.py#L868, I'll definitely add comments here.